spec/index.html

```diff
@@ -367,6 +367,39 @@ <h2>Introduction</h2>
       <p>There is also a [[[SPARQL12-RESULTS-XML]]] which follows a similar design pattern but uses XML as the
       serialization.</p>
       <p>Unless otherwise noted in the section heading, all sections and appendices in this document are normative.</p>
+
+	  <section id="version-announcement">
+	    <h3>Version Announcement</h3>
+	    <p>To indicate that RDF 1.2 features are used in the SPARQL results,
+        such as <a data-cite="RDF12-CONCEPTS#dfn-base-direction">initial text direction</a> or <a data-cite="RDF12-CONCEPTS#dfn-triple-term">triple terms</a>,
+	    the optional `version` <a href="#media-type">Media Type parameter</a> can be set to `1.2`.
```

Comment:
Maybe use normative wording?
```suggestion
	    the optional `version` <a href="#media-type">Media Type parameter</a> MAY be set to `1.2`.
```